apps/server/src/infra/etherpad-client/interface/error-status.enum.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorType {
```

Comment:
I think "EtherpadErrorType" would be a better name. And please align the file name to "etherpad-error-type.enum.ts".